### PR TITLE
Fix stats by country in nginx status page

### LINKED ISSUE #1194: Nginx Ingress Controller - Stats by country does not work
Hi,

the latest stats by country does not work. When i go to /nginx_status page there aren't any filters by country.

Thank you.

### PR DESCRIPTION
**Pull request by aledbf**
captura de pantalla 2016-06-17 a las 6 27 05 p m

fixes #1194

### CODE REVIEW

**Comment by aledbf** [comment]
ping @bprashanth

File: ingress/controllers/nginx/nginx.tmpl [hunk 1]
```diff
@@ -16,18 +16,24 @@ events {
 }
 
 http {
+    {{/* we use the value of the header X-Forwarded-For to be able to use the geo_ip module */}}
+    real_ip_header      X-Forwarded-For;
```
Comment (bprashanth) [inline ingress/controllers/nginx/nginx.tmpl]: for my own education, can it geo locate based on src ip as well?

**Comment by bprashanth** [comment]
LGTM

File: ingress/controllers/nginx/nginx.tmpl (hunk 1 above)
Comment (aledbf) [inline ingress/controllers/nginx/nginx.tmpl]: yes that's the default if no `X-Forwarded-For` is present in the request

**Pull request merged**